bindings/cs/MongoDB.Crypt.Test/BasicTests.cs

```diff
@@ -260,6 +260,27 @@ public void EncryptExplicitStepwise()
             }
         }
 
+        [Fact]
+        public void TestDataKeyCreation()
+        {
+            var altKeyNames = new[] {"KeyMaker", "Architect"};
+            var altKeyNameDocuments = altKeyNames.Select(name => new BsonDocument("keyAltName", name));
+            var altKeyNameBuffers = altKeyNameDocuments.Select(BsonUtil.ToBytes);
+            var key = new LocalKmsCredentials(new byte[96]);
+            var keyId = new LocalKeyId(altKeyNameBuffers);
```

Comment:
Can/should we also test with AwsKeyId?

Reply:
Yes! Pushed up a new test

Reply:
Pushed up a stepwise version of the same test as well